fixtures/raml/api.raml

```diff
@@ -0,0 +1,48 @@
+#%RAML 1.0
+title: Hello world
+
+/helloworld:
+  get:
+    responses:
+      200:
+        body:
+          application/json:
+            type: !include types/response.json
+            example: !include examples/get-response.json
+  post:
+    body:
+      application/json:
+        type: !include types/response.json
+        example: !include examples/post-response.json
+    responses: 
+      201:
+        body:
+          application/json: 
+            type: !include types/response.json
+            example: !include examples/post-response.json
+  /{id}:
+    put:
```

Comment:
Don't know if this is relevant in the scope of this ticket but I will put it here anyway.

If a put request occurs and there is not a current representation on the server, then the put request is to return a 201. If the target resource does have a representation you need to send a 200 or 204.

[https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT#Responses](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT#Responses)